### PR TITLE
stm32f746g-disco: Move serial console from USART6 to USART1 (Virtual …

### PR DESCRIPTION
**Pull request by acassis**
## Summary
Move serial console from USART6 to USART1
## Impact
User will be able to use the Virtual COM port directly
## Testing
N/A

### CODE REVIEW

File: boards/arm/stm32f7/stm32f746g-disco/README.txt [hunk 1]
```diff
@@ -98,16 +98,20 @@ LEDs and Buttons
...
   -------- ---------------
               STM32F7
-  ARDUIONO FUNCTION  GPIO
-  -- ----- --------- -----
-  DO RX    USART6_RX PC7
-  D1 TX    USART6_TX PC6
-  -- ----- --------- -----
+  V.COM     FUNCTION  GPIO
+  -----    --------- -----
+  RXD      USART6_RX PB7
```
Comment (ghn-certi) [inline boards/arm/stm32f7/stm32f746g-disco/README.txt:108]:
```suggestion
  RXD      USART1_RX PB7
```

File: boards/arm/stm32f7/stm32f746g-disco/README.txt [hunk 2]
```diff
@@ -98,16 +98,20 @@ LEDs and Buttons
...
+  TXD      USART6_TX PA9
```
Comment (ghn-certi) [inline boards/arm/stm32f7/stm32f746g-disco/README.txt:109]:
```suggestion
  TXD      USART1_TX PA9
```

**Comment by acassis** [comment]
Thank you Gustavo!

**Comment by Ouss4** [comment]
There are other places in the README where USART6 is still referenced.
Like the one under NOTES, STATUS and Configuration Directories/nsh (it's called UART6 there)

**Comment by Ouss4** [comment]
@acassis please squash the commits so we can merge it.

**Comment by acassis** [comment]
> @acassis please squash the commits so we can merge it.

Done, thank you Abde!

**Pull request merged**